### ISSUE TITLE
compatibility with V8 greater than v7.6 (patch for building)

### ISSUE THREAD

**Issue by artynet** [theturtle32/WebSocket-Node#373]
Hello @theturtle32,

please take a look at my compatibility patch here : 47d16204fce6311447388adb0191a59f882612ac

It fixes the build with the latest release of node v12 :
```
make: Entering directory '/root/WebSocket-Node/build'
  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
../src/bufferutil.cc: In static member function 'static Nan::NAN_METHOD_RETURN_TYPE BufferUtil::Merge(Nan::NAN_METHOD_ARGS_TYPE)':
../src/bufferutil.cc:57:59: warning: 'v8::Local<v8::Value> v8::Object::Get(uint32_t)' is deprecated: Use maybe version [-Wdeprecated-declarations]
       Local<Object> src = Nan::To<v8::Object>(array->Get(i)).ToLocalChecked();
                                                           ^
In file included from /root/.cache/node-gyp/12.13.0/include/node/v8-internal.h:14:0,
                 from /root/.cache/node-gyp/12.13.0/include/node/v8.h:25,
                 from ../src/bufferutil.cc:8:
/root/.cache/node-gyp/12.13.0/include/node/v8.h:3461:51: note: declared here
   V8_DEPRECATED("Use maybe version", Local<Value> Get(uint32_t index));
                                                   ^
/root/.cache/node-gyp/12.13.0/include/node/v8config.h:311:3: note: in definition of macro 'V8_DEPRECATED'
   declarator __attribute__((deprecated(message)))
   ^~~~~~~~~~
  SOLINK_MODULE(target) Release/obj.target/bufferutil.node
  COPY Release/bufferutil.node
  CXX(target) Release/obj.target/validation/src/validation.o
  SOLINK_MODULE(target) Release/obj.target/validation.node
  COPY Release/validation.node
make: Leaving directory '/root/WebSocket-Node/build'
```
this is the output of :
```
$ node-gyp rebuild
```
I need to fix this deprecation warning and then I'm done. Would you mind giving me some pointers about it ? The NAN entry to fix is this one :
```
Local<Object> src = Nan::To<v8::Object>(array->Get(i)).ToLocalChecked();
```

Kind Regards

**Comment by artynet** [comment]
@theturtle32 this should fix the latest deprecation warning:
```
Local<Object> src = Nan::To<v8::Object>(Nan::Get(array, Nan::New(i)).ToLocalChecked()).ToLocalChecked();
```
I am ready to submit a pull request if you want. Please double check my final code...

**Comment by theturtle32** [comment]
Fixed in release v1.0.31 on npm.
